Sources/KlaviyoSDKDependencies/AnyCodable/AnyCodable.swift

```diff
@@ -0,0 +1,150 @@
+/// Adapted from AnyCodable v0.6.7 on 11/15/2024
+/// https://github.com/Flight-School/AnyCodable/tree/0.6.7
+/// Comments - modified to make this @unchecked Sendable
+
+import Foundation
+/**
+ A type-erased `Codable` value.
+
+ The `AnyCodable` type forwards encoding and decoding responsibilities
+ to an underlying value, hiding its specific underlying type.
+
+ You can encode or decode mixed-type values in dictionaries
+ and other collections that require `Encodable` or `Decodable` conformance
+ by declaring their contained type to be `AnyCodable`.
+
+ - SeeAlso: `AnyEncodable`
+ - SeeAlso: `AnyDecodable`
+ */
+@frozen public struct AnyCodable: Codable, @unchecked Sendable {
+    public let value: Any
+
+    public init<T>(_ value: T?) {
+        self.value = value ?? ()
+    }
+}
+
+extension AnyCodable: _AnyEncodable, _AnyDecodable {}
+
+extension AnyCodable: Equatable {
```

Comment:
very suprised this isn't natively supported in swift. Is this something that we do get with structs but not necessarily codables? Makes sense that we need it for state management operations but I'm just surprised to see it so explicit. Does this mean we need to manually add any new types supported from the backend?

Reply:
See my later comment.